src/ui/public/config/config.js

```diff
@@ -119,7 +119,10 @@ any custom setting value for "${key}" may fix this issue.
 You can also save an step using \`config.get("${key}", defaultValue)\`, which
 will set the initial value if one is not already set.`);
```

Comment:
This last bit of the sentence needs to change to something like

> will default to the supplied value when it is unset instead of throwing this error.